pkg/modelregistry/jsonvalues/convertjson.go

```diff
@@ -109,9 +109,9 @@ func CorrectJSONPaths(jsonBase string, jsonPathValues []*change.ConfigValue,
 				break
 
 			} else if hasPrefixMultipleIdx(jsonPathWildIndex, modelPathWildIndex) {
-				jsonSubPath := jsonPathWildIndex[len(modelPathWildIndex):]
+				//jsonSubPath := jsonPathWildIndex[len(modelPathWildIndex):]
```

Comment:
why was this commented out ? if it's not needed please remove

Reply:
removed